woodwork/deserialize.py

```diff
@@ -57,6 +59,43 @@ def _typing_information_to_woodwork_table(table_typing_info, validate, **kwargs)
     kwargs = loading_info.get('params', {})
     table_type = loading_info.get('table_type', 'pandas')
 
+    logical_types = {}
+    semantic_tags = {}
+    column_descriptions = {}
+    column_metadata = {}
+    use_standard_tags = {}
+    dtypes = {}
```

Comment:
We only add categorical dtypes to this, right? Can we call this something like `dtype_categories` or `category_cols`?

Reply:
Updated name